gammapy/scripts/image_pipe.py

```diff
@@ -16,6 +20,16 @@
 class ImageAnalysis(object):
     """Gammapy 2D image based analysis.
 
+    Fill a `~gammapy.image.SkyMapCollection`:
```

Comment:
is it what you want?
How to have new line between each maps description in the docstring?